khal/controllers.py

```diff
@@ -21,6 +21,8 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
So basically my point is that changing literals everywhere (by importing `unicode_literals`) might create regressions for current Python 2 users, which are arguably the majority right now.